src/igvfd/schemas/auxiliary_set.json

```diff
@@ -1,6 +1,6 @@
 {
     "title": "Auxiliary Set",
-    "description": "A file set for auxiliary raw data files that were produced alongside raw data files from a measurement set. For example, in a CRISPR screen experiment the measurement set would capture the result of single-cell transcript sequencing and the auxiliary set the result of gRNA sequencing with the associated cellular barcodes.",
+    "description": "AuxiliarySet was designed to capture files (FASTQs) that result from sequencing of nucleic acids from a sample that are a proxy to some vital information that would allow you to analyze the experiment. Auxiliary sets usually would not tell you anything about the transcriptome or the genome of the sample in question. For example AuxiliarySet would include the sequencing of barcodes that correspond to elements introduced into cells, or sequencing of guide RNA coding sequences in the cells. The files in the AuxiliarySet have to do with the type of experiment and are relevant for the Analysis, but they by themselves are not telling you much about biology in the sample associated with AuxiliarySet.",
```

Comment:
This applies to all 3 descriptions here, but I think generally in the schema JSONs we have avoided referring to schemas without spaces in the words. So usually we say "in an auxiliary set [...]" not "In AuxiliarySets".

Reply:
I would probably also change "but they by themselves are not telling you much about biology in the sample" to: "but by themselves do not tell much about the biology of the sample". I think using the word "you" in the description makes it sound less formal.